src/lib/mongoBackend/mongoSubscribeContextAvailability.cpp

```diff
@@ -99,6 +98,7 @@ HttpStatusCode mongoSubscribeContextAvailability
     }
     sub.append(CASUB_ATTRS, attrs.arr());
 
+    // FIXME P5: RenderFormat right now hardcoded to "JSON" (NGSI_V1_LEGACY), in the future the RenderFormat will be taken from the payload
     /* Adding format to use in notifications */
     sub.append(CASUB_FORMAT, "JSON");
```

Comment:
Maybe it would be more correct using `getFormatString(NGSI_V1_LEGACY)` instead of `JSON` literal? 

(I don't remember right now the rigth name of the getFormatString function)